lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -1739,6 +1739,41 @@ Error TFLiteModelLoader::loadConv2D(const tflite::Operator *op,
                       opErrMsg(opInfo, "Filter must be constant!"));
     RETURN_ERR_IF_NOT(llvm::dyn_cast<Constant>(bias.getNode()),
                       opErrMsg(opInfo, "Bias must be constant!"));
+
+    // Modify quantization parameters, if the model is wrong quantized.
+    Constant *filterC = llvm::dyn_cast<Constant>(filter);
+    Constant *biasC = llvm::dyn_cast<Constant>(bias);
+    auto filterScalesH = filterScales->getHandle<float>();
+    auto filtersH = filterC->getHandle<int8_t>();
+    auto biasH = biasC->getHandle<int32_t>();
+    auto biasScalesH = biasScales->getHandle<float>();
+    RETURN_ERR_IF_NOT(
+        filterScalesH.size() == biasScalesH.size(),
+        opErrMsg(opInfo,
+                 "Number of filter scales must be equal with bias scales."));
+    for (int idx = 0; idx < filterScalesH.size(); idx++) {
+      if (filterScalesH.raw(idx) == 0 && biasScalesH.raw(idx) == 0) {
+        auto start =
+            idx * filter.dims()[1] * filter.dims()[2] * filter.dims()[3];
+        auto frame = filter.dims()[1] * filter.dims()[2] * filter.dims()[3];
+        for (int jdx = start; jdx < (start + frame); jdx++) {
+          filtersH.raw(jdx) = 0;
+        }
+        biasH.raw(idx) = 0;
+        filterScalesH.raw(idx) = 0.125;
```

Comment:
Nit: it would be nice to add a comment here about this value.

Reply:
A comment has been added on top of the section with more details.